pyproject.toml

```diff
@@ -51,6 +56,16 @@ exclude = '''
 )
 '''
 
+[tool.isort]
```

Comment:
Please use config from https://github.com/evrone/python-codestyle

Reply:
ok, I will

Reply:
but i afraid to get isort/black cnoflicts... will see